### DISCUSSION TITLE
2.0.1

### DISCUSSION THREAD

**Discussion by joschi** [OpenAPITools/openapi-diff#312] [Announcements]
## What's Changed
* Enable reproducible builds by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/300
* Skip Git hook repository check to enable build without `.git` by @joschi in https://github.com/OpenAPITools/openapi-diff/pull/301

**Full Changelog**: https://github.com/OpenAPITools/openapi-diff/compare/2.0.0...2.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenAPITools/openapi-diff/releases/tag/2.0.1'>2.0.1</a>.</em>

**Comment by julienrf** [comment]
Hello!
The main branch contains important bug fixes, would it be possible to cut a new release soon?
Thank you
